recipes/snakemake-executor-plugin-slurm/meta.yaml

```diff
@@ -24,8 +24,8 @@ requirements:
   run:
     - python >=3.11.0,<4.0.0
     - snakemake-interface-common >=1.13.0,<2.0.0
-    - snakemake-interface-executor-plugins >=9.0.0,<10.0.0
-    - snakemake-executor-plugin-slurm-jobstep >=0.1.10,<0.2.0
+    - snakemake-interface-executor-plugins >=9.1.1,<10.0.0
+    - snakemake-executor-plugin-slurm-jobstep >=0.2.0,<0.3.0
```

Comment:
Looks like this is from https://github.com/snakemake/snakemake-executor-plugin-slurm/pull/66/files which was merged only after the `v0.4.3` tag.